ecms/src/main/java/org/exoplatform/platform/qa/ui/ecms/pageobject/SiteExplorerHome.java

```diff
@@ -1601,4 +1604,39 @@ public void verifyWebContentInformationCreatedSuccessfully(String Content){
   public enum defineValueBehavior {
     CREATE_NEW, REMOVE_EXISTING, REPLACE_EXISTING, THROW_EXEPTION;
   }
+
+
+  /**
+   * Click on View Document button
+   */
+  public void clickViewDocumentButton() {
+    $(byText("View Document")).waitUntil(Condition.visible,Configuration.openBrowserTimeoutMs).click();
+  }
+
+
+  /**
+   * Click on Edit Online button
+   */
+  public void clickEditOnine() {
+    $(byText("Edit Online")).click();
```

Comment:
Elements should not be selected by text, but rather by id or classes.

Reply:
ok